pom.xml

```diff
@@ -76,6 +76,12 @@
             <version>2.6.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
shouldn't need to add this, it comes via the nucleus dependency chain

Reply:
yeah was trying w/o it at first but my IDE wouldn't recognize, maybe i missed something

Reply:
If you remove, does it still compile?

Reply:
Yeah I removed component-common from my maven cache and did a maven build and it worked, and looks like it compiled fine in GitHub too